examples/tcp-close/client.rs

```diff
@@ -294,39 +292,6 @@ impl TcpClient {
         self.qds.insert(qd);
         Ok(qd)
     }
-
-    /// Issues a close() operation and deregisters the queue descriptor.
-    fn issue_close(&mut self, qd: QDesc) -> Result<()> {
-        if self.should_async_close {
-            let qt: QToken = self.libos.async_close(qd)?;
-
-            match self.libos.wait(qt, None) {
-                Ok(qr) if qr.qr_opcode == demi_opcode_t::DEMI_OPC_CLOSE && qr.qr_ret == 0 => (),
-                Ok(qr) if qr.qr_opcode == demi_opcode_t::DEMI_OPC_FAILED && is_closed(qr.qr_ret) => (),
-                Ok(_) => anyhow::bail!("wait() should succeed with async_close()"),
-                Err(_) => anyhow::bail!("wait() should succeed with async_close()"),
-            }
-        } else {
-            match self.libos.close(qd) {
-                Ok(_) => (),
-                Err(e) if e.errno == libc::ECONNRESET => (),
-                Err(_) => anyhow::bail!("wait() should succeed with close()"),
-            }
-        }
-        self.qds.remove(&qd);
```

Comment:
We are now missing to remove the closed 'qd' from 'qds'.

Reply:
Done.

Reply:
Done.